pkg/apis/serving/v1alpha1/revision_defaults_test.go

```diff
@@ -130,6 +130,41 @@ func TestRevisionDefaulting(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "lemonade",
+		wc:   v1beta1.WithUpgradeViaDefaulting,
+		in: &Revision{
+			Spec: RevisionSpec{
+				DeprecatedContainer: &corev1.Container{
+					Image: "foo",
+					VolumeMounts: []corev1.VolumeMount{{
+						Name: "bar",
+					}},
+				},
+				RevisionSpec: v1beta1.RevisionSpec{
+					ContainerConcurrency: 1,
```

Comment:
If you set ContainerConcurrency (or the other fields) in both places which one wins? Can we get a test?